health_checks.js

```diff
@@ -0,0 +1,103 @@
+/* -----------------------------------------------------------------------------
+ * @copyright (C) 2018, Alert Logic, Inc
+ * @doc
+ *
+ * Halth check functions common for all collectors
+ *
+ * @end
+ * -----------------------------------------------------------------------------
+ */
+
+const AWS = require('aws-sdk');
+const async = require('async');
+
+/**
+ * @function
+ *
+ * @param {Object} event - checkin event for Lambda function, can be used to send needed parameters for health checks. Example: stackName.
+ * @param {Object} context - context of Lambda's function.
+ * @param {function} callback - callback, which is called by health check when it's done.
+ *
+ * @returns {(null|ErrorMsg)} returns null if all OK or ErrorMsg if something was wrong
```

Comment:
@kkuzmin not sure if it's really correct way to put it with `@returns`, as actually `callback` is called on return... do you have an idea how it's usually  being done with jsDoc?

Reply:
Good example is here https://github.com/Azure/azure-sdk-for-node/blob/master/lib/services/keyVault/lib/keyVault.js#L44-L56